src/StreamJsonRpc.Shared/Reflection/JsonRpcMethodAttribute.cs

```diff
@@ -4,7 +4,7 @@
 namespace StreamJsonRpc
 {
     /// <summary>
-    /// Attribute which allows a method to have an alternate name.
+    /// Attribute which allows a method to have a different name.  If applied on a method, the method's CLR name can no longer be used for remote invocation.
```

Comment:
consider: "allows a method to have a different name" -> "changes the name by which this method can be invoked over JSON-RPC." Your clarifying second sentence sounds great.